racingcar-java/src/test/java/step1/StringTest.java

```diff
@@ -0,0 +1,44 @@
+package step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+    @Test
+    @DisplayName("문자열 분리")
+    void splitTest() {
+        String[] result = "1,2".split(",");
+        assertThat(result).contains("1");
+    }
+
+    @Test
+    @DisplayName("문자열 분리, 원소가 정확히 일치")
+    void splitExactlyTest() {
+        String[] result = "1,2".split(",");
+        assertThat(result).containsExactly("1","2");
+    }
+
+    @Test
+    @DisplayName("문자열 자르기")
+    void substringTest() {
+        String result = "(1,2)".substring(1, 4);
+        assertThat(result).isEqualTo("1,2");
+    }
+
+    @Test
+    @DisplayName("특정 위치의 문자 가져오기")
+    void charAtTest() {
+        String string = "abc";
+        char result = string.charAt(0);
+        assertThat(result).isEqualTo('a');
```

Comment:
```suggestion
        // given 
        String string = "abc";
        
        // when 
        char result = string.charAt(0);
        
        // then
        assertThat(result).isEqualTo('a');
```
숙달된 개발자 및 개발 문화면 작성하지 않아도 되지만  
이번 미션은 연습이 목적이므로 다음과 같은 BDD 형태를 최대한 가져가주시면 좋을 것 같아요